data/databases.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE courses_course 
+(
+    id INT PRIMARY KEY NOT NULL,
+    key VARCHAR(255),
+),
+CREATE TABLE auth_user 
+(
+    id INT PRIMARY KEY NOT NULL, 
+    user_id INT, 
+    org VARCHAR(255),
+    course_id VARCHAR(255),
+    role VARCHAR(255),
+),
+CREATE TABLE auth_user
+(
+    id INT PRIMARY KEY NOT NULL, 
+    email NVARCHAR(255), 
+),
```

Comment:
Don't forget to also add records related to ES entries (course key) and instructors.

Reply:
I got confused between the names of the tables, two of them was named `auth_user`. Has been fixed